cluster-autoscaler/main.go

```diff
@@ -292,6 +292,7 @@ func buildAutoscaler() (core.Autoscaler, error) {
 		processors.NodeGroupSetProcessor = &nodegroupset.BalancingNodeGroupSetProcessor{
 			Comparator: nodegroupset.IsAzureNodeInfoSimilar}
 	}
+	processors.ScaleDownNodeProcessor = core.NewDefaultScaleDownNodeProcessor()
```

Comment:
Shouldn't that happen in ca_processors.DefaultProcessors()? It does for all other processors..

Reply:
Was put in to avoid cyclic dependency. Now that the processor is not in core, I moved this.